src/providers/coinstore.ts

```diff
@@ -0,0 +1,52 @@
+import { asNumber, asObject, asString, asTuple, asValue } from 'cleaners'
+import fetch from 'node-fetch'
+
+import { config } from '../config'
+import { NewRates, RateMap, ReturnRate } from '../rates'
+import { logger } from '../utils/utils'
+
+const { uri } = config.providers.coinstore
+
+const asCoinstoreResponse = asObject({
+  data: asTuple(
+    asObject({
+      id: asValue(922),
+      symbol: asValue('LLDUSDT'),
+      price: asString
+    })
+  ),
+  code: asNumber
+})
+
+// For convenience, this plugin assumes USDT === iso:USD
+
+const coinstoreRateMap = (
+  results: ReturnType<typeof asCoinstoreResponse>
+): RateMap => ({ 'LLD_iso:USD': results.data[0].price })
+
+const coinstore = async (
+  requestedRates: ReturnRate[],
+  currentTime: string
+): Promise<NewRates> => {
+  const rates = { [currentTime]: {} }
+
+  // This is a LLD-only provider so we can check to exit early
+  if (requestedRates.find(req => req.currency_pair === 'LLD_iso:USD') == null) {
```

Comment:
Looks like the rate is actually against USDT. Since there's always a chance for a depeg, this should just be LLD_USDT and then let the route finder get the USDT to USD rate.